testutil/network/network.go

```diff
@@ -199,7 +200,48 @@ func SupplierModuleGenesisStateWithAccounts(t *testing.T, addresses []string) *s
 	return state
 }
 
-// Initialize an Account by sending it some funds from the validator in the network to the address provided
+// ApplicationModuleGenesisStateWithAccounts generates a GenesisState object with
+// an application list full of applications with the given addresses.
+func ApplicationModuleGenesisStateWithAccounts(t *testing.T, addresses []string) *apptypes.GenesisState {
```

Comment:
We already have `DefaultApplicationModuleGenesisState`, so I don't think copy-pasting that is correct.

Why not consolidate the two?

Reply:
Remnants from my old test strategy
removing